### DISCUSSION TITLE
How to Join the Kubescape Slack Channel

### DISCUSSION THREAD

**Discussion by mukesh-mahato** [kubescape/kubescape#1310] [General]
I have been trying to find information on how to join the Kubescape Slack channel, as I believe it would be a great platform to engage with fellow developers and seek guidance on various topics. However, I couldn't find any specific instructions or links to join the Slack channel on the official GitHub repository or website.
Could someone kindly guide me through the process of joining the Kubescape Slack channel? I am eager to connect with like-minded individuals, share ideas, and collaborate on Kubernetes projects.

**Comment by craigbox** [comment]
https://kubescape.io/project/community/#slack
